lib/crawler/data/crawl_result/html.rb

```diff
@@ -124,6 +124,38 @@ def meta_description(limit: 1024)
           Crawler::ContentEngine::Utils.limit_bytesize(description, limit)
         end
 
+        def meta_tags_elastic(limit: 1024)
```

Comment:
Nit: I think the default should be the same as the config default

```suggestion
        def meta_tags_elastic(limit: 512)
```